contributed_definitions/NXcxi_ptycho.nxdl.xml

```diff
@@ -62,12 +62,10 @@
           Application definition for a ptychography experiment. This is compatible with CXI from version 1.6 (discussed with Filippe Maia) if this application definition
```

Comment:
Is this text necessary for the documentation?

> (discussed with Filippe Maia)

Reply:
I've now removed this text.